### ISSUE TITLE
Error setup Kafka provider

### ISSUE THREAD

**Issue by phlucas96** [apache/incubator-openwhisk-devtools#271]
During the installation of the kafka provider, docker-compose tries to pull the openwhisk/controller:"latest" image instead of the "nightly" image, giving an error:

```
Pulling controller (openwhisk/controller:latest)...
ERROR: manifest for openwhisk/controller:latest not found
```

This is because during the "setup-providers" phase of the Makefile, the image prefix and image tag are missing from the "providers.env" file. 

Adding the following lines to the Makefile "setup-providers" phase fixes the problem.

```
printf "DOCKER_IMAGE_PREFIX"="openwhisk\n" >> $(TMP_HOME)/tmp/openwhisk/providers.env

printf "DOCKER_IMAGE_TAG"="nightly\n" >> $(TMP_HOME)/tmp/openwhisk/providers.env
```

**Comment by rabbah** [comment]
@phlucas96 are you on master/head?

**Comment by rabbah** [comment]
Thanks @phlucas96 I see the delta. Do you want to submit a PR?

**Comment by phlucas96** [comment]
Ok, I will. Thanks.

**Comment by rabbah** [comment]
closed by #272
